src/loader/bgw_interface.c

```diff
@@ -57,3 +62,9 @@ ts_bgw_db_workers_restart(PG_FUNCTION_ARGS)
 {
 	PG_RETURN_BOOL(bgw_message_send_and_wait(RESTART, MyDatabaseId));
 }
+
+Datum
+ts_bgw_loader_api_version(PG_FUNCTION_ARGS)
+{
+	PG_RETURN_INT32(LOADER_API_VERSION);
```

Comment:
I think it would be better to make this a rendezvous variable, or at least both.

We should also consider making the API version an exported `const` variable (not only function). This might allow inspecting that variable (e.g., using `objdump`) without actually having to load the shared object.